### DISCUSSION TITLE
How to change request address in SwaggerForOcelot?

### DISCUSSION THREAD

**Discussion by KuS233** [Burgyn/MMLib.SwaggerForOcelot#260] [Q&A]
I have a multiservice project in kubernetes. In the ingress, I set the website address to "http://domain/api". I have access to the swagger site. After changing DownstreamSwaggerEndPointBasePath I have access to shared services. Unfortunately, the request address is set to "http://domain/", which causes me to get a 404 error when I invoke the request. Can I somehow change the request address in SwaggerForOcelot?

**Comment by Burgyn** [comment]
Hi,

can you please share some minimal reproducible example?

Thanks.

**Reply by KuS233** [reply]
Hi,
Thank you for your answer. Below is the configuration of SwaggerForOcelot in startup.cs:

            application.UseSwaggerForOcelotUI(opt =>
            {
                opt.DownstreamSwaggerEndPointBasePath = "/api/swagger/docs" ;
            });

            application.UseSwagger();

            application
                .UseSwaggerUI(c =>
                {
                    c.SwaggerEndpoint($"./{ApiVersion}/swagger.json", ApiName);
                });


Everything looks like it's working: 

![1](https://user-images.githubusercontent.com/11790371/226212752-ba5effe2-8907-469a-ad94-49f7b1eb2859.png)

I can see all the services and controllers that I have made available in ocelot. Unfortunately, when I try to execute any of the swagger endpoints, I get a 404.: 

![2](https://user-images.githubusercontent.com/11790371/226212613-2a93abfd-1436-4705-a6ac-726aab415167.png)

**Reply by nzdjigit** [reply]
Hi there! Did you manage to find a solution to this? We've run into the exact same problem and struggling to find a solution to this.
